mozregression/bisector.py

```diff
@@ -351,6 +351,51 @@ def handle_merge(self):
         return result
 
 
+"""
+We are just using this to make it clear we have no merge to take care of
+We are running an Integration because builds are triggered from cron jobs
+on mozilla-central for all Snap package branches
+"""
+
+
+class SnapHandler(IntegrationHandler):
+    snap_repo = None
+    _build_infos = {}
+    snap_rev = {}
+
+    def __init__(self, **kwargs):
+        super(IntegrationHandler, self).__init__(**kwargs)
+
+    def record_build_infos(self, build_infos):
```

Comment:
I did the best I could, but it may not yet be clear enough because I dont properly remember the way the caller code works (I remember it was far from being trivial to follow)